201-web-app-asev2-ilb-create/azuredeploy.json

```diff
@@ -28,23 +50,27 @@
             }
         },
         "internalLoadBalancingMode": {
-            "type": "int",
+            "type": "string",

```

Comment:
the type, default and allowed values will not work together

Reply:
Updated the default value to string